chaos-test/chaostest/chaostest/aws/aws_wrapper.py

```diff
@@ -0,0 +1,162 @@
+                    dest='aws_az',
+                    help="AWS aplication zone under usage")
+parser.add_argument("-chaosFile",
+                    action=ChaosAction,
+                    default="ec2-delete.json",
+                    dest="file",
+                    help="Chaos file to chose for execution"
+                    )
+parser.add_argument("-boto3Resource",
+                    action=ChaosAction,
+                    required=False,
+                    dest="aws_resource",
+                    default="ec2-iks",
+                    help="Chose aws resource to initialize boto3")
+parser.add_argument("-verifySSL",
+                    required=False,
+                    default="false",
+                    action=ChaosAction,
+                    dest='KUBERNETES_VERIFY_SSL',
+                    help="Kubernetes client ignore SSL")
+parser.add_argument("-kubeConfig",
+                    required=False,
+                    default="",
+                    dest='kubeconfig',
+                    help="Kubernetes client ignore SSL")
+parser.add_argument("-kubeContext",
+                    required=False,
+                    default=os.environ.get("KUBECONFIG", ""),
+                    dest='kubeContext',
+                    help="Kubernetes client ignore SSL")
+parser.add_argument('-report', action=ChaosAction,
+                    dest='report',
+                    default="false",
+                    help='Option to upload the result to report server')
+parser.add_argument('-report_endpoint', action=ChaosAction,
+                    dest='report_endpoint',
+                    default="none",
+                    help='Endpoint where the report kubernetes will be uploaded')
+parser.add_argument("-namespace", action=ChaosAction,
+                    required=False,
+                    default="dev-reliabilityiks3-usw2-qal",
+                    dest="name_space",
+                    help="namespace for application"
+                    )
+parser.add_argument("-experiment", action=ChaosAction,
+                    required=False,
+                    dest="exp",
+                    default="experiment-aws",
+                    help="Experiment yaml name"
+                    )
+parser.add_argument("-testNamespace",
+                    required=False,
+                    default="dev-reliabilityiks3-usw2-qal",
+                    dest='test_namespace',
+                    help="Namespace on which chaos results will be persisted")
+parser.add_argument('-label', action=ChaosAction,
+                    dest='label_name',
+                    default="app",
+                    help='Store a label value')
+parser.add_argument('-app', action=ChaosAction,
+                    required=False,
+                    dest='app_endpoint',
+                    default="localhost",
+                    help='Store the application health endpoint')
+parser.add_argument('-chaosrole-name', action=ChaosAction,
```

Comment:
this variable not reflect correctly, either make the session as variable name or handle it differently.

Reply:
Done